pkg/payload/payload.go

```diff
@@ -245,9 +245,11 @@ func GetImplicitlyEnabledCapabilities(updatePayloadManifests []manifest.Manifest
 				continue
 			}
 			if err := currentManifest.Include(nil, nil, nil, &clusterCaps); err != nil {
-				continue
+				break
 			}
 			if err := updateManifest.Include(nil, nil, nil, &clusterCaps); err != nil {
+				klog.V(2).Infof("%s has changed and is now part of one or more disabled capabilities which will be implicitly enabled: %v",
+					getManifestResourceId(updateManifest), err)
```

Comment:
I feel like this is asking us to create a `Manifest.String()` method in library go, as follow-up work ;)